admin-portal/src/pages/supplierQuotationUpload/supplierQuotationUpload.tsx

```diff
@@ -79,13 +79,20 @@ export function SupplierQuotationUpload() {
         <div className="uploadSupplierQuotationContainer">
           <form onSubmit={handleSubmit}>
             <p>Select the needs csv file that you want to upload.</p>
-            <input type="file" accept=".csv" onChange={handleChange} />
+            <input
+              type="file"
+              accept=".csv"
+              onChange={handleChange}
+              value={fileName}
+            />
             <button type="submit">Upload</button>
           </form>
         </div>
         <div className="error-list-div">
-          {errorList.length > 0 &&
-            errorList.map((error: any) => <p>{error}</p>)}
+          {responseData.length > 0 &&
+            responseData.map((error: string, index: number) => (
```

Comment:
We should avoid using the index as a key. What we can do instead is generate a unique key before setting it to the state. 
Read more:
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Or else we can just display the error payload as it is (with line breaks). 
ex: https://stackoverflow.com/questions/39325414/line-break-in-html-with-n

Reply:
I prefer the second option.

Reply:
i split the string cuz the success message has vertical bars separating the lines. i'll ask anupama to use line breaks instead. i changed the code. pls review.